### ISSUE TITLE
Handling of <q> element in output

### ISSUE THREAD

**Issue by raducoravu** [dita-ot/dita-ot#2481]
In the XHTML output by default text inside the "q" element is surrounded in dumb double quotes.
But in the PDF output the same text is surrounded in smart double quotes.
Maybe we should have consistency in these outputs.

**Comment by shaneataylor** [comment]
There exist `OpenQuote` and `CloseQuote` strings (`xsl/common/strings-en-us.xml`), which I assume are being used for XHTML and to facilitate both internationalization and house style preferences for "straight" or "curly" quotes. Are you saying we should simply change the default values for some languages to the curly quote symbols?

**Comment by shaneataylor** [comment]
...and that assumption seems to have been incorrect. 😦

**Comment by jelovirt** [comment]
This should be fixed in SASS.

**Comment by raducoravu** [comment]
Right now as @shaneataylor says the "strings-en-us.xml" generates actual characters for those elements. So @jelovirt, are you suggesting giving up the  "strings-en-us.xml" approach for this case and style the spans via the CSS?

**Comment by robander** [comment]
Long ago when these quote strings were added + used in XHTML, the browsers did not have reliable formatting for `<q>` (some showed quotes, some did not, and I think there were also issues with localization). 

I tested this a couple of months ago when somebody's validation tool complained about the quotes, and all of the browsers I tried with correctly displayed quotes for something like `<q>test</q>` in the HTML. I think it's time to stop pulling in the string variables for X/HTML and instead just use the native `<q>` element.
